FirebaseAppCheck/Tests/Unit/AppAttestProvider/FIRAppAttestProviderTests.m

```diff
@@ -797,6 +923,48 @@ - (void)assertGetToken_WhenKeyRegistered_Success {
   [self verifyAllMocks];
 }
 
+- (void)expectAppAttestAvailabilityToBeCheckedAndStoredKeyRequested {
```

Comment:
It wasn't immediately obvious to me that this would be setting up a rejection as part of the expectation - is there a way we can encode that in the (already long) name?